rules.mk

```diff
@@ -133,7 +119,8 @@ define DOCKERFILE_RULE
 	    -e 's|ARG_ARCH|$(ARCH)|g' \
 	    -e 's|ARG_BIN|$(BINARY)|g' \
 	    -e 's|ARG_REGISTRY|$(REGISTRY)|g' \
-	    -e 's|ARG_FROM|$(BASEIMAGE)|g' \
+	    -e 's|ARG_FROM_BASE|$(BASEIMAGE)|g' \
+	    -e 's|ARG_FROM_IPT|$(IPTIMAGE)|g' \
```

Comment:
Is this because an if condition checking BINARY to be "node-cache" isn't matching?

Reply:
I think the if that i've initially put only see empty BINARY variable

Also this fix make it clear that we have 2 base images
Tell me if you prefer any other names for the variables
If the fix seems ok to you I can squash the commits